src/wallet/wallet.cpp

```diff
@@ -983,12 +983,12 @@ void CWallet::MarkDirty()
 {
     {
         LOCK(cs_wallet);
-        for (std::pair<const uint256, CWalletTx>& item : mapWallet)
+        for (std::pair<const Txid, CWalletTx>& item : mapWallet)
             item.second.MarkDirty();
```

Comment:
structured bindings nit
```suggestion
        for (auto& [_, wallet_tx] : mapWallet)
            wallet_tx.MarkDirty();
```

Reply:
Done. I love structured bindings now.